molecule/scenario_with_local_binary_directory/converge.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: "Include ansible-role-promtail"
+      include_role:
+        name: "ansible-role-promtail"
```

Comment:
As with the other scenarios, this extra converge steps shouldn't be required. Did you encounter any issues that were resolved by adding this definition?

Reply:
Indeed, I have just removed this file and note that it is not necessary. I thought he was. as much for me, I withdraw it.